src/Components/WebAssembly/DevServer/src/build/Microsoft.AspNetCore.Components.WebAssembly.DevServer.targets

```diff
@@ -2,6 +2,6 @@
   <PropertyGroup>
     <_BlazorDevServerDll>$(MSBuildThisFileDirectory)../tools/blazor-devserver.dll</_BlazorDevServerDll>
     <RunCommand>dotnet</RunCommand>
-    <RunArguments>&quot;$(_BlazorDevServerDll)&quot; serve --applicationpath &quot;$(TargetPath)&quot;</RunArguments>
+    <RunArguments>&quot;$(_BlazorDevServerDll)&quot; --applicationpath &quot;$(TargetPath)&quot;</RunArguments>
```

Comment:
Any idea if there are other tools (e.g. the debugger) that rely on the "serve" command? We can add it back, but it seems fairly unnecessary.

Reply:
Debugger doesn't rely on serve so I think this change is sensible. I always thought having the command was excessive but figured there was a good reason for it. Good to know we can simplify things here!

Reply:
When we first made the dev server we had imaginings of it doing a whole bunch of different things in time, hence the commands. Clearly that's not how it panned out, so yeah - it's nice that you're simplifying it away.